CodeSniffer/Standards/Generic/Sniffs/PHP/DisallowAlternativePHPTagsSniff.php

```diff
@@ -0,0 +1,253 @@
+<?php
+/**
+ * Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff.
+ *
+ * PHP version 5
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+/**
+ * Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff.
+ *
+ * Verifies that no alternative PHP tags are used.
+ *
+ * If alternative PHP open tags are found, this sniff can fix both the open and close tags.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @author    Marc McIntyre <mmcintyre@squiz.net>
+ * @copyright 2006-2014 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ * @version   Release: @package_version@
+ * @link      http://pear.php.net/package/PHP_CodeSniffer
+ */
+class Generic_Sniffs_PHP_DisallowAlternativePHPTagsSniff implements PHP_CodeSniffer_Sniff
+{
+
+
+    /**
+     * Whether ASP tags are enabled or not.
+     *
+     * @var bool
+     */
+    private $_asp_tags = false;
```

Comment:
Good point (though the phpcs code style check does not trigger on it - maybe a rule should be added to the project's own phpcs.xml file ?)
Anyway, fixed it. Let me know if you'd like me to squash the commits.

(am working with three different sniff projects at the moment using different code styles, so this was a left over from one of the others :sunglasses: )